RUFAS/routines/animal/pen.py

```diff
@@ -14,166 +14,105 @@
     optimize as growing_heifer_optimize
 from RUFAS.routines.animal.ration import cow_requirements as req
 from RUFAS.routines.animal.ration import ration_driver as ration_driver
+import copy
 
 
 class Pen:
     """
     Manages a pen's operation. Stores the characteristics of the pen and the
     animals that are housed in it any point in the simulation. This class also
     keeps track of some key characteristics of the average animal in the pen.
+
+    Attributes
+    -------------
+    id : int
+        This variable represents a pen's unique pen ID, gotten from the input file.
+
+    vertical_dist_to_parlor : float
+        This variable represents the vertical distance to milking parlor, measured in kilometers.
+        It is gotten from the input file.
```

Comment:
```suggestion
        Obtained from the input file.
```
small language clarity suggestion